src/html/popup.html

```diff
@@ -45,6 +45,14 @@
 
         <main>
             <div class="container">
+                <div id = "rateSection">
+                    <div class="section">
```

Comment:
Once the user clicks on "Rate it now" button the whole div including <div class="section"> and <div class="divider"></div> should be hidden.